themes/bootstrap3/templates/librarycards/home.phtml

```diff
@@ -52,6 +52,9 @@
   <?php endif; ?>
 
   <div class="btn-group">
+    <?php if($this->shibboleth): ?>
```

Comment:
I think this needs to be configurable and default to being disabled. In most cases the option would probably just confuse users. While adding configuration I'd suggest adding a comment on why and how the feature is used, and if any there are any prerequisites e.g. in Shibboleth configuration.